documentation/how-to/gnss.en.md

```diff
@@ -214,4 +214,6 @@ If an averaged position minimum count requirement is active, a progress bar will
 To setting to activate an average position minimum count threshold can be found in QField settings's *positioning* panel.
 When active, holding the add vertex button is not required, a short tap on the button will begin the collection of positions and automatically add the averaged position when the minimum count requirement is met.
 
-!![](../assets/images/positioning_averaged_set.jpg)
\ No newline at end of file
+!![](../assets/images/positioning_averaged_set.jpg)
+
+In this case, when using [`@gnss_*` variables](../gnss/#additional-variables) it will give the mean of all collected points.
```

Comment:
@nirvn can you review this sentence (one thing I cannot answer is gnss or position variables)

Reply:
That's an accurate statement

Reply:
Just to be crystal clear to our doc readers, that means `gnss_` variabls are averaged, `position_` variables are not averaged?

Reply:
Na, both would be averaged.

Reply:
@lucienicolier can you add that?